index.bs

```diff
@@ -999,7 +1013,7 @@ an [=attribution source=] |source|:
 1. If |source|'s [=attribution source/debug reporting enabled=] is false, return null.
 1. Let |body| be a new [=map=] with the following key/value pairs:
     : "`attribution_destination`"
-    :: |source|'s [=attribution source/attribution destination=].
+    :: |source|'s [=attribution source/attribution destinations=].
```

Comment:
This should use the algorithm you defined as "serialize [=event-level report/attribution destinations=]"